salmiakki/src/components/LeaderboardPage.js

```diff
@@ -0,0 +1,7 @@
+export function LeaderboardPage() {
+    return (
+        <p>
+            <img src="https://s.france24.com/media/display/7e96fbec-8829-11ea-bb5d-005056a964fe/w:1280/p:16x9/NORTHKOREA-POLITICS-KIM.webp" width="100%" />
```

Comment:
Maybe not point image urls to outside sources.

Reply:
Temporarily. We are not having kim jong un memes later on.